test/e2e-kind.sh

```diff
@@ -20,33 +20,35 @@ set -euo pipefail
 
 KOURIER_GATEWAY_NAMESPACE=kourier-system
 KOURIER_CONTROL_NAMESPACE=knative-serving
+CLUSTER_SUFFIX=${CLUSTER_SUFFIX:-cluster.local}
 
 export KO_DOCKER_REPO=kind.local
```

Comment:
nit:
`KO_DOCKER_REPO` is defined in `.github/workflows/kind-e2e.yaml`. Should we remove this line or define it only when it isn not defined like `CLUSTER_SUFFIX`?